qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/files/03-profiles.py

```diff
@@ -263,7 +263,9 @@ def configure_user(username, groups, uid=1000, gid=100):
             # mount the shared directories for user only if there are
             # shared folders (groups) that the user is a member of
             # else ensure that the `shared` folder symlink does not exist
-            f"ln -sfn /shared /home/{username}/shared" if groups else f"rm -f /home/{username}/shared",
+            f"ln -sfn /shared /home/{username}/shared"
+            if groups
+            else f"rm -f /home/{username}/shared",
```

Comment:
`rm -f /home/{username}/shared` condition will no longer occur.

Reply:
I guess it's safe to have this condition either way. So no need to remove

Reply:
I don't think that's true - the admins are free to remove `analyst` as a default group in theory, or who knows what else... as you say, either way it's safe.

Reply:
BTW not sure if Black versions have changed somewhere because it's only a formatting change here!